### ISSUE TITLE
Selecting tank immediately gives it a destination

### ISSUE THREAD

**Issue by mlms13** [mlms13/april-game#10]
This is because clicks fall through and don't get trapped.  I think @fponticelli solved this in ledge, maybe?

**Comment by fponticelli** [comment]
Yes he did. The trick is to use the new feature in "update" that allows you to return a bool value. If `true` the update cycle falls through to the next system. If `false` it stops there.
Since this might stop more than just the system you want to cancel, it is suggested to create a new sub-phase and group the systems in there. In fact the update cancellation happens at the phase level.
